inference-engine/src/vpu/myriad_plugin/myriad_plugin.cpp

```diff
@@ -52,7 +53,10 @@ ExecutableNetworkInternal::Ptr Engine::LoadExeNetworkImpl(
         manager.run_passes(function);
 
         ngraph::pass::Manager ti_manager;
+        // Apply all transformations to TensorIterator body
         ti_manager.register_pass<ngraph::pass::ApplyTransformationsToTIBody>(manager);
+        // Unroll should be called after all conversions
+        ti_manager.register_pass<ngraph::pass::UnrollTensorIterator>();
```

Comment:
Is it a new functionality for Myriad? I would expect that you remove something and insert this one